modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -84,6 +84,34 @@ test.describe('Manage object definitions through Model Builder', () => {
 
 		// Clean up
 
+		await apiHelpers.objectAdmin.deleteObjectDefinition(
+			objectDefinition.id
+		);
+	});
+	test('show object definition details in "RightSidebar" after create object definition', async ({
```

Comment:
Hey @rafaellajordao, to fix the SF error:
> consecutive test blocks should be separated by a blank line

We need to add a blank line here.

Reply:
I was unable to use SF locally, is there any way to do this? :disappointed: